tests/e2e-kfddistribution-6-migrate-from-nginx-to-none.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+# shellcheck disable=SC2154
+
+load ./helper
+
+@test "Ingress Nginx NS has been deleted" {
```

Comment:
Wishlist: it would be nice to check that the infra ingresses in the other namespaces are deleted too.

Reply:
Yes, let's do that later